framework/src/modules/network/network.js

```diff
@@ -200,7 +204,9 @@ module.exports = class Network {
 
 		this.p2p.on(EVENT_REQUEST_RECEIVED, async request => {
 			this.logger.info(
```

Comment:
This one should be trace too because it's very often.
Also `EVENT_MESSAGE_RECEIVED` is similar so it should be trace.